Runtime/Behaviours/SetUpBuilding.cs

```diff
@@ -6,10 +6,12 @@
 
 namespace ReupVirtualTwin.behaviours
 {
-    public class SetUpBuilding : MonoBehaviour , ISetUpBuilding
+    public class SetUpBuilding : MonoBehaviour , ISetUpBuilding, IOnBuildingSetup, IBuildingGetter
     {
         [SerializeField]
-        GameObject building;
+        GameObject _building;
```

Comment:
```suggestion
        private GameObject _building;
```
I would expect this to be private.

Reply:
it's supposed to be private by default, you mean you want to see the word `private` in the code?